src/builtins/filters/array.rs

```diff
@@ -282,6 +277,11 @@ pub fn slice(value: &Value, args: &HashMap<String, Value>) -> Result<Value> {
         end = arr.len();
     }
 
+    // Not an error, but returns an empty Vec
+    if start >= end {
+        return Ok(Vec::<Value>::new().into());
+    }
+
```

Comment:
Can you add a test for it?

Reply:
Done!

BTW, I tried adding `(make_args(Some(-1), Some(-4.0)), vec![])` to tests but it gave compile error "error[E0600]: cannot apply unary operator `-` to type `usize`".